hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -29,6 +30,8 @@
   // TODO (elingg) better handling in livestate and persistent state of zkfc task. Right now they are
   // chained.
   private Task zkfcNodeTask;
+  //Timed Health Check for node health monitoring
+  private Timer healthCheckTimer;
```

Comment:
If we are going to have health checks in the abstract base... shouldn't the healthCheckTimer be protected in the base?  the code is the same in both executors.  The only difference is the creation of the the scheduled health check.

I would even be inclined to add a function in the base for `scheduleHealthCheck(...)`  

This Timer and the initialization are the same in both executors.